libmultilabel/metrics.py

```diff
@@ -69,7 +69,7 @@ def eval(self):
         }
         # add metrics like P@k, R@k to the result dict
         scores = precision_recall_at_ks(y_true, y_pred, top_ks=self.top_ks)
-        precision_recall_monitored = set(self.monitor_metrics) & scores.keys()
+        precision_recall_monitored = sorted(set(self.monitor_metrics) & scores.keys())
```

Comment:
One tricky example, if we want to monitor ``P@1, P@5, P@15``, it will return ``P@1, P@15, P@5`` in default sort setting.
Maybe we could use 
```python
precision_recall_monitored = [metric for metric in self.monitor_metrics if metric in scores]
```
so that we can achieve set intersection and maintain original order

Reply:
@henryyang42  fixed